src/components/card/Card/Card.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface CardProps {
+  header?: React.ReactNode
+  media?: React.ReactNode
+  footer?: React.ReactNode
+  standardLayout?: 'default' | 'headerFirst'
+  flagLayout?: 'default' | 'mediaOnRight'
+}
+
+export const Card = (
+  props: CardProps & React.HTMLAttributes<HTMLDivElement>
+): React.ReactElement => {
+  const {
+    header,
+    media,
+    footer,
+    standardLayout,
+    flagLayout,
+    children,
+    className,
+    ...divProps
+  } = props
+
+  // TODO determine how to implement the tablet:grid-col-4 class properly, should this be li class?
+  const classes = classnames(
+    'usa-card',
+    {
+      'usa-card--header-first': standardLayout == 'headerFirst',
```

Comment:
We should be enforcing `===` for equality operators... I'm a little surprised it doesn't seem to be in our eslint config. we should look into that

Reply:
🤦